project/recursive_sorting.py

```diff
@@ -44,9 +44,24 @@ def merge_sort_in_place(arr, l, r):
 
 # TO-DO: implement the Quick Sort function below
 def quick_sort( arr, low, high ):
+    
+    if len(arr) < 2:
+        return arr
 
+    pivot = arr[0]
+    left = []
+    right = []
+    for i in range(1, len(arr)):
+        if arr[i] < pivot:
+            left.append(arr[i])
```

Comment:
Really good implementation Alejandro but for your logic, also have the smaller indexes increment by one for accuracy